modules/openy_features/openy_location/modules/openy_loc_camp/openy_loc_camp.module

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * @file
+ * OpenY Camp module file.
+ */
+
+/**
+ * Implements hook_theme_suggestions_page_alter().
+ */
+function openy_loc_camp_theme_suggestions_page_alter(array &$suggestions, array $variables) {
+  // Add content type suggestion for Landing pages related to a camp.
+
+  // Exit if not a node page, or if the 'openy_loc_camp.camp_service' is not available.
+  if (!($node = \Drupal::request()->attributes->get('node')) || !($camp_service = \Drupal::service('openy_loc_camp.camp_service'))) {
```

Comment:
No need to do that atm, but as a follow-up - It is better not to use service word in service machine names )